01_Generation/00_generate_w_bedrock.ipynb

```diff
@@ -181,8 +181,8 @@
     "- `amazon.titan-tg1-large`\n",
     "- `ai21.j2-grande-instruct`\n",
     "- `ai21.j2-jumbo-instruct`\n",
-    "- `anthropic.claude-instant-v1`\n",
-    "- `anthropic.claude-v1`"
+    "- `anthropic.claude-instant-v2`\n",
```

Comment:
Is there a v2 available for the instant model? I don't see it on Anthropic's main site, let alone on Bedrock.

Please confirm.

Reply:
Thanks. I'm getting the exact model names from a teammate now.

Reply:
Will do a find/replace once I've determined what they are.